app/shared/modals/reservation-success/ReservationSuccessModal.js

```diff
@@ -74,15 +76,18 @@ function ReservationSuccessModal({
               </p>
               <p>
                 {t('ReservationSuccessModal.ownReservationsPageHelpText')}
-                {email &&
+                {email
+                  && (
                   <span>
                     {' '}
                     <FormattedHTMLMessage
                       id="ReservationSuccessModal.emailHelpText"
                       values={{ email }}
                     />
                   </span>
-                }.
+                  )
+                }
+.
```

Comment:
This will look better if it's embedded in `ReservationSuccessModal.emailHelpText` message.